### ISSUE TITLE
[Bug]: 508 Accessibility Compliance: Missing Form Label

### ISSUE THREAD

**Issue by jeran-urban** [orchidjs/tom-select#997]
### Bug description

I am using version 2.5.1:

When you initiate tom-select, it hides the existing select and takes over the associated label (updating the for to point to the internal input). If you use accessibility testing tools (like WAVE for chrome), you can see this causes an error where the existing select is missing a form label now. 

Error:
*Missing form label*

my current custom workaround

```
function addMissingFormLabel(selectId) {
    const select = document.getElementById(selectId);
    const labelText = $(select).prevAll('label').first().text();

    const hiddenLabel = document.createElement('label');
    hiddenLabel.setAttribute('for', selectId);
    hiddenLabel.className = 'visually-hidden'; //bootstrap class
    hiddenLabel.textContent = labelText;

    select.insertAdjacentElement('beforebegin', hiddenLabel);
}

//add hidden label for compliance in init callback
onInitialize: function () {
    addMissingFormLabel('my-select-id');
},
```

### Expected behavior

The package leaves code accessible, which in this case would involve either hiding the existing label and creating a custom one for the newly added input; or after repurposing the label, create a new hidden one that will leave the code compliant for automated testing.

### Steps to reproduce

simply initiate a new tom select. You can see existing examples by loading the [WAVE accessibility chrome extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) and running it against the [tom-select examples home page](https://tom-select.js.org/examples/).

### Additional context

- OS: Windows
- Browser chrome
- Version (chrome: 144.0.7559.135) (tom-select npm: 2.5.1)
- Device: cpu

**Comment by jeran-urban** [comment]
Hello,

I would agree adding another label is not a perfect solution, but it is a simple one that maintains compliance. The aria-hidden attribute is helpful, however 508 (WCAG compliance) validation technologies still pick up the finding since technically it is not accessible to have an input without a label. To be clear that was the first thing I tried (including display none and other options to try to make it ignored).

**Comment by NicolasCARPi** [comment]
@jeran-urban It works if you add `aria-label` to your select element, see example here: https://github.com/orchidjs/tom-select/pull/999

**Comment by jeran-urban** [comment]
@NicolasCARPi , thank you! aria-label worked like a charm, various compliance software scans are reporting with no issues. Thank you!

**Comment by afagard** [comment]
Someone explain to me how `aria-label` is *actually* helping here. The select box contents are completely removed by TS. Compliance software scans might find this acceptable but it actually isn't right? Screen readers will just see an empty select.

**Comment by xJuvi** [comment]
@afagard 
Please Show us screenshots and a working code example. This is required to reproduce a bug if exists.
